# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1363314203 14400
#      Thu Mar 14 22:23:23 2013 -0400
# Node ID f299079ed03a57ddd0db3a4ee4a6dbc87ff4a1b2
# Parent  4a3b6a80db2e6631dbbfbc678998d05230d991ae
avoid shadowed variable warning from GCC

* file-editor.cc (file_editor::handle_file_name_changed): Rename
arguments to avoid shadowed variable warning from GCC.

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -426,27 +426,28 @@ file_editor::mru_menu_update ()
     // save actual mru-list in settings
     QSettings *settings = resource_manager::get_settings ();
     // FIXME -- what should happen if settings is 0?
     settings->setValue ("editor/mru_file_list",_mru_files);
     settings->sync ();
 }
 
 void
-file_editor::handle_file_name_changed (const QString& fileName, const QString& toolTip)
+file_editor::handle_file_name_changed (const QString& fname,
+                                       const QString& tip)
 {
   QObject *fileEditorTab = sender();
   if (fileEditorTab)
     {
       for(int i = 0; i < _tab_widget->count (); i++)
         {
           if (_tab_widget->widget (i) == fileEditorTab)
             {
-              _tab_widget->setTabText (i, fileName);
-              _tab_widget->setTabToolTip (i, toolTip);
+              _tab_widget->setTabText (i, fname);
+              _tab_widget->setTabToolTip (i, tip);
             }
         }
     }
 }
 
 void
 file_editor::handle_tab_close_request (int index)
 {
